util/design/gen-flash-img.py

```diff
@@ -106,27 +187,57 @@ def main(argv: List[str]):
                 data = int(item, 16)
                 # `data_w_intg_ecc` will be in format {ECC bits, data bits}.
                 data_w_intg_ecc, _ = secded_gen.ecc_encode(
-                    config, "hamming", FLASH_WORD_SIZE, data)
+                    ecc_configs, "hamming", FLASH_WORD_SIZE, data)
                 # Due to storage constraints the first nibble of ECC is dropped.
                 data_w_intg_ecc &= 0xFFFFFFFFFFFFFFFFF
```

Comment:
Maybe we could break this constant up with underscores to make the bit positions more obvious: `0xF_FFFFFFFF_FFFFFFFF` ?

Or, would it be easier to understand as `2**n - 1` ?

Reply:
done.